backend/official_plugins/tracim_backend_autoinvite/__init__.py

```diff
@@ -35,7 +35,7 @@ def on_user_created(self, user: User, context: TracimContext) -> None:
                     user=user,
                     workspace=workspace,
                     role_level=workspace.default_user_role.level,
-                    email_notification_type=EmailNotificationType.SUMMARY,
+                    email_notification_type=EmailNotificationType.DAILY,
```

Comment:
I cans ee multiple line where `EmailNotificationType.DAILY` is used as default value. I suggest to create and use a `classfunction` like this : 

```python
class EmailNotificationType(enum.Enum):
    INDIVIDUAL = "individual"
    NONE = "none"
    SUMMARY = "summary"
    HOURLY = "hourly"
    DAILY = "daily"
    WEEKLY = "weekly"

    @classfunction
    def default(cls) -> "EmailNotificationType":
        return cls.DAILY
```

Reply:
Will fix